modules/rostests/apitests/rtl/amd64/RtlCaptureContext.c

```diff
@@ -167,14 +153,54 @@ START_TEST(RtlCaptureContext)
     ok_eq_hex64(CapturedContext.Legacy[6].High, OriginalContext.Legacy[6].High & 0xFF);
     ok_eq_hex64(CapturedContext.Legacy[7].Low, OriginalContext.Legacy[7].Low);
     ok_eq_hex64(CapturedContext.Legacy[7].High, OriginalContext.Legacy[7].High & 0xFF);
+#else
+    ok_eq_hex(CapturedContext.FltSave.ControlWord, 0xcccc);
```

Comment:
These are all left uninitialized in kernel mode? :eyes:

Reply:
yes